modules/home/assets/js/components/sidebar-promotion.js

```diff
@@ -0,0 +1,39 @@
+import { Box, Paper, Typography, Stack } from '@elementor/ui';
+import Button from '@elementor/ui/Button';
+import List from '@elementor/ui/List';
+import ListItem from '@elementor/ui/ListItem';
+import ListItemText from '@elementor/ui/ListItemText';
+import SideBarCheckIcon from '../icons/side-bar-check-icon';
```

Comment:
empty line

```suggestion

import SideBarCheckIcon from '../icons/side-bar-check-icon';
```

Reply:
why? we don't have it in other files? all imports are one after the other

Reply:
@elchugreeva You are right.
You can ignore my comment.
We have only added the empty lines for components, not for icons.